index.bs

```diff
@@ -1018,6 +1038,32 @@ associated.
         a user's avatar or a [=relying party=]'s logo.
 </div>
 
+
+### Authenticator Selection Criteria ### {#authenticatorSelection}
+
+The {{AuthenticatorSelectionCriteria}} dictionary describes a set of criterias for which the [RPS] can use to find the appropriate
+authenticator 
+
+<xmp class="idl">
+    dictionary AuthenticatorSelectionCriteria {
+        Attachment    attachment = false;
```

Comment:
`false` isn't a valid [`Attachment`](https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/AngeloKai/webauthn/angelo-secondAuthnrSelect/index.bs#enumdef-attachment) value, so it can't be the default here. I think you want to leave this field without a default, since the algorithm treats not-present differently from either present value.